tests/unit/test_help.py

```diff
@@ -117,6 +119,23 @@ def test_can_page_output_on_windows(self):
         renderer.render('foo')
         self.assertEqual(renderer.popen_calls[-1][0], (['more'],))
 
+    @unittest.skipIf(platform.system() not in ['Darwin', 'Linux'],
+                    "Ctrl-C not valid on windows.")
+    def test_can_handle_ctrl_c(self):
+        class CtrlCRenderer(FakePosixHelpRenderer):
+            def _popen(self, *args, **kwargs):
+                if self._is_pager_call(args):
+                    os.kill(os.getpid(), signal.SIGINT)
+                return self.mock_popen
+
+            def _is_pager_call(self, args):
+                return 'less' in args[0]
+
+        renderer = CtrlCRenderer()
+        renderer.mock_popen.communicate.return_value = ('rendered', '')
+        renderer.exists_on_path['groff'] = True
+        renderer.render('foo')
+
```

Comment:
Good catch, it's missing an assertion.  I'll assert that the rendered input is sent to the pager despite receiving a ctrl-c.